test/playground.test.ts

```diff
@@ -5,16 +5,21 @@ import {Tab} from '../src/sdk';
 import {PLAYGROUND_ORIGIN} from '../src/constants';
 import {PluginSdk} from '../src/plugin';
 
-jest.mock('../src/constants', () => ({
-    PLAYGROUND_CONNECT_URL: 'https://play.croct.com/connect.html',
-    PLAYGROUND_ORIGIN: 'https://play.croct.com',
-}));
-
+jest.mock(
+    '../src/constants',
+    () => ({
+        PLAYGROUND_CONNECT_URL: 'https://play.croct.com/connect.html',
+        PLAYGROUND_ORIGIN: 'https://play.croct.com',
+    }),
+);
+
+// eslint-disable-next-line jest/require-top-level-describe -- Don't repeat this hook in each describe
```

Comment:
I don't think we should keep the beforeEach on the top level. IMO we should repeat for each `describe`.